frontend/src/redux/slices/position-slice.ts

```diff
@@ -70,6 +70,22 @@ export const positionSlice = createAppSlice({
         },
       },
     ),
+    uploadPositionsFromCsv: create.asyncThunk(
```

Comment:
Tbh there's no need for this to go through Redux (with our current api setup which leaves much to be desired), since it's not actually doing anything with the state. 

You don't have to fix this, but just a note.